main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	golog "log"
+	"os"
+	"testing"
+
+	componenttest "github.com/ONSdigital/dp-component-test"
+	"github.com/ONSdigital/dp-frontend-release-calendar/features/steps"
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/cucumber/godog"
+	"github.com/cucumber/godog/colors"
+)
+
+var componentFlag = flag.Bool("component", false, "perform component tests")
+
+func InitializeScenario(ctx *godog.ScenarioContext) {
```

Comment:
got some shadowing of ctx here. Recommend upgrading the linter and it'll complain :D